db/migrate/20190606175619_create_targets.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTargets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :targets do |t|
+      t.text :title
+      t.references :user, foreign_key: true
+      t.integer :length
```

Comment:
why do you need the length of the targets in a column 🤔

Reply:
Thi is the radius of the target